### PR TITLE
discuss issues

### PR DESCRIPTION
**Pull request by nikomatsakis**
This refines the workflow discussion and incorporates the new issue templates.

[Rendered preview](https://nikomatsakis.github.io/wg-async-foundations/)

### CODE REVIEW

File: src/vision/status_quo/grace_deploys_her_service.md [hunk 1]
```diff
@@ -7,6 +7,12 @@
 
 [Grace deploys her service and is able to fix problems]: ./shiny_future.md#grace-deploys-her-service-and-is-able-to-fix-problems
 
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstormind period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today.
```
Comment (lqd) [inline src/vision/status_quo/grace_deploys_her_service.md:12]:
```suggestion
This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today.
```

File: src/vision/status_quo/template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# 😱 Status quo stories: Template
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstormind period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today.
```
Comment (lqd) [inline src/vision/status_quo/template.md:5]:
```suggestion
This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
```

**Comment by lqd** [comment]
Seems pre-existing but the two markdown blocks at the top of `src/vision/status_quo/alan_runs_into_stack_trouble.md` seem to trigger CI errors. mdbook thinks they are tests and marking their code block's start as 
```
```ignore
```

[here](https://github.com/rust-lang/wg-async-foundations/blame/7a39f212c81af120676d0ef05c049748f3231eea/src/vision/status_quo/alan_runs_into_stack_trouble.md#L14-L24) should fix it.

**Comment by nikomatsakis** [comment]
@lqd yeah I fixed it, thanks!

**Pull request merged**